src/components/FilterDrawer/FilterDrawer.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import Drawer from 'antd/lib/drawer';
+
+import DateRangeForm from '../DateRangeForm/DateRangeForm';
+
+class FilterDrawer extends React.Component {
+	render() {
+		return (
+			<div>
+				<Drawer
+					title="Filter Results by Time Range"
+					placement="right"
+					closable={false}
+					onClose={this.props.handleDrawer}
+					visible={this.props.drawerVisible}
```

Comment:
could this state and controlling functions go in this component? Keeping logic as close to where it's used

Reply:
nvm, I see it's used in NavBar too